src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -194,6 +214,8 @@ public static void main(String[] args) {
    */
   private Map<String, String> importRenameMap = Collections.emptyMap();
 
+  private Map<String, String> aliasMap = Collections.emptyMap();
```

Comment:
Describe the need for aliasMap here or point to the documentation in the other class.

Reply:
Done.